### PR TITLE
[Opt](Vectorized) Support push down no grouping agg

### PR DESCRIPTION
**Pull request by HappenLee**
# Proposed changes

Use the segment static to speed up some not grouping agg query：`min`, `max`, `count`

Here need to a new SegmentIter call `StatisticsIter` to do the work.

lineorder : 60kw line num

query             Before             After
count(*)         0.5s                0.02s
min                0.4s                0.01s
max               0.4s                 0.01s

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/olap/rowset/segment_v2/column_reader.cpp [hunk 1]
```diff
@@ -171,6 +171,44 @@ Status ColumnReader::get_row_ranges_by_zone_map(
...
+    bool is_string = is_olap_string_type(type);
+    if (max_value->is_null()) {
+        assert_cast<vectorized::ColumnNullable&>(*dst).insert_default();
+    } else {
+        if (is_string) {
+            auto sv = (StringValue*)max_value->cell_ptr();
+            dst->insert_data(sv->ptr, sv->len);
+        } else {
+            dst->insert_many_fix_len_data(static_cast<const char*>(max_value->cell_ptr()), 1);
+        }
```
Comment (BiteTheDDDDt) [inline be/src/olap/rowset/segment_v2/column_reader.cpp:191]: why not just use `insert_data` here?

File: be/src/olap/rowset/segment_v2/column_reader.cpp [hunk 2]
```diff
@@ -171,6 +171,44 @@ Status ColumnReader::get_row_ranges_by_zone_map(
...
+        assert_cast<vectorized::ColumnNullable&>(*dst).insert_null_elements(size);
+    } else {
+        if (is_string) {
+            auto sv = (StringValue*)min_value->cell_ptr();
+            dst->insert_many_data(sv->ptr, sv->len, size);
+        } else {
+            // TODO: the work may cause performance problem, opt latter
+            for (int i = 0; i < size; ++i) {
+                dst->insert_many_fix_len_data(static_cast<const char*>(min_value->cell_ptr()), 1);
+            }
```
Comment (BiteTheDDDDt) [inline be/src/olap/rowset/segment_v2/column_reader.cpp:205]: Maybe we can use `insert_many_fix_len_data( ,size)`

Comment (HappenLee) [reply be/src/olap/rowset/segment_v2/column_reader.cpp:205]: en，just add a todo here to support in the future

File: be/src/olap/rowset/segment_v2/column_reader.cpp (hunk 1 above)
Comment (HappenLee) [reply be/src/olap/rowset/segment_v2/column_reader.cpp:191]: may have uint24_t use in `Date` , `insert_data` may cause bug.

**Review by BiteTheDDDDt — DISMISSED** [review]
LGTM

**Review by BiteTheDDDDt — APPROVED** [review]
LGTM

**Pull request merged**